.github/policies/resourceManagement.yml

```diff
@@ -170,6 +170,23 @@ configuration:
         
         - [ ] Complete [localizability testing](https://github.com/NuGet/NuGet.Client/blob/dev/docs/ui-guidelines.md#localizability-testing).'
       description: 'Remind PR author to test accessibility and localizability'
+    - if:

```

Comment:
As commented in another PR: https://github.com/NuGet/NuGet.Client/pull/6426#discussion_r2069655269

we don't need to put all the bot policies in one file. It's easier to maintain if different policies are in different files.

Reply:
I'm aware, and I chose to put this PR file path rule with the other file path rules because I prefer that. It looks like half the team has approved and as I mentioned elsewhere, it's extremely important to me to have people remember to consider servicing branches in their CI. We can completely rethink the file split of bot rules in this issue https://github.com/NuGet/Client.Engineering/issues/3267

Reply:
We should definitely break into different files! Thank you Donnie for the new tracking issue.